blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -40,6 +40,9 @@ Breaking Changes
 Changes
 =======
 
+- Expand the ``search_path`` setting to accept a list of schemas that will be
+  searched when a relation is referenced without specifying a schema.
```

Comment:
I think we could/should also mention that `pg_catalog` is now also implicitly searched for relations.